app/views/public/api_documentation.html.erb

```diff
@@ -208,6 +208,11 @@
   <div class="col-sm-12">
     <p>The Epoch endpoint will return all epochs data. Replace `:network` with 'testnet' or 'mainnet' as desired. </p>
     <p>Request:</p>
+    <pre>curl -H "Token: secret-api-token" 'https://www.validators.app/api/v1/epoch/:network/index.json'</pre>
```

Comment:
I made some minor improvements and that is no longer the endpoint, please check `rails routes`

Reply:
I've already fixed this in previous commit.